File: For Fun/Assignment 1/C# version/TelephoneNumber.cs
using System;

class TelephoneNumber
{
	public static void Main(string[] args)
	{
		Random ran = new Random();
		Console.WriteLine("Random telephone number: 0" + +ran.Next(10) + ran.Next(10) + ran.Next(10) + "-"
						  + (ran.Next(9) + 1) + ran.Next(10) + ran.Next(10) + ran.Next(10) + ran.Next(10)
						  + ran.Next(10));
	}
}

File: Assignment 1/Java version/TelephoneNumber.java
import java.util.Random;

public class TelephoneNumber {

	public static void main(String[] args) {
		Random ran = new Random();
		System.out.println("Random telephone number: 0" + ran.nextInt(10) + ran.nextInt(10) + ran.nextInt(10) + "-"
				+ (ran.nextInt(9) + 1) + ran.nextInt(10) + ran.nextInt(10) + ran.nextInt(10) + ran.nextInt(10)
				+ ran.nextInt(10));
	}
}
